### PR TITLE
Fixed filename too long and empty image file errors, removed redundant example notebooks

### LINKED ISSUE #28: Filename too long error
On Linux

image

This happened with the following artworks:
[artwork 1](https://artsandculture.google.com/asset/maestrapeace-detail-of-lesbian-couple-mural-on-the-san-francisco-women-s-building-18th-and-valencia-streets-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez-%C2%A9-1994-2000-2010-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez/cgEsw3X5NVcwXw)
[artwork 2](https://artsandculture.google.com/asset/maestrapeace-lapidge-street-fa%C3%A7ade-mural-on-the-san-francisco-women-s-building-18th-and-valencia-streets-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez-%C2%A9-1994-2000-2010-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez/EgGfNVyaG8m0Tg)
[artwork 3](https://artsandculture.google.com/asset/maestrapeace-corner-view-mural-on-the-san-francisco-women-s-building-18th-and-valencia-streets-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez-%C2%A9-1994-2000-2010-juana-alicia-edythe-boone-miranda-bergman-susan-cervantes-meera-desai-yvonne-littleton-and-irene-perez/AQHbiHz-OyvNdQ)

### PR DESCRIPTION
**Pull request by modhurita**
Hi @qubixes , @jgarciab :

I have fixed the issues mentioned in the title. Could you please review my code and accept the pull request if everything looks ok?

Fixes #28 .

Thanks,
Modhurita

### CODE REVIEW

File: artscraper/googleart.py [hunk 1]
```diff
@@ -45,29 +45,32 @@ def load_link(self, link):
...
         # Prevent problems with character encoding/decoding
         paint_id = unquote(paint_id)
-        # Prevent problems with too-long file/directory names
-        if len(paint_id)>=256:
-            paint_id = paint_id[0:255]
 
-        return Path(self.output_dir, paint_id)
+        # Prevent problems with too-long file/directory names
+        path_str = str(Path(self.output_dir, paint_id))
+        if len(path_str)>=256:
```
Comment (qubixes) [inline artscraper/googleart.py:69]: If I understand correctly, this gives a directory, right? What is the problem exactly? The /x/y/z, where the whole thing is too long, or only if x or x or z are too long?

What if the output_dir path is already too long?

File: artscraper/googleart.py [hunk 2]
```diff
@@ -164,11 +172,13 @@ def save_image(self, img_fp=None, link=None):
 
         img_fp = self._convert_img_fp(img_fp, suffix=".png")
 
-        if self.skip_existing and img_fp.is_file():
+        if self.skip_existing and img_fp.is_file() and img_fp.stat().st_size!=0:
```
Comment (qubixes) [inline artscraper/googleart.py:175]: Is it really a size of 0? If I remember correctly, it was an actual image, just completely white, but maybe that has changed.

File: examples/example_collect_all_artworks.ipynb
Comment (qubixes) [inline examples/example_collect_all_artworks.ipynb:1]: I think you might want to clear the outputs, just in case.

**Review by qubixes — CHANGES_REQUESTED** [review]
Just some small questions regarding how it  fixes the issues. Otherwise, nice work in cleaning up the notebooks.

File: artscraper/googleart.py (hunk 1 above)
Comment (modhurita) [reply artscraper/googleart.py]: I fixed this using SHA-1 hash.

**Comment by modhurita** [comment]
Hi @qubixes : 

I have made the changes you requested. Does it look okay now?

**Comment by modhurita** [comment]
Hi @qubixes:

I have now made the change you requested yesterday (set filename truncation length to 240).

**Review by qubixes — APPROVED** [review]
I approve the changes!

**Pull request merged**